library/option.js

```diff
@@ -37,39 +63,80 @@
             } catch (e) { }
 
             option.getAsync(key).then(function (val) {
+				//console.log(key,val);
                 if (val == null) {
                     val = val_default;
                     option.set(key, val);
                 }
                 if (val === val_checked) {
                     checkbox.prop('checked', true);
-                }
+                }else{
+					checkbox.prop('checked', false);
+				}
                 //console.log(val, val_checked, val_unchecked);
-
+				_auto_login_setting(key,val);
                 checkbox.change(function () {
                     //console.log('changed');
+					//console.log(key);
                     let val = checkbox.prop('checked') ? val_checked : val_unchecked;
+					//console.log(val);
                     option.set(key, val)
                         .then(function () {
                             if (chrome.runtime.lastError) {
                                 console.log(chrome.runtime.lastError);
                             }
                         });
+					_auto_login_setting(key,val);
                 });
             });
         });
     }
 
+	function initInput() {
+		option.getAsync('usr').then(function (usr) {
+			$('#usr').val(usr);
+		});
+		option.getAsync('pwd').then(function (pwd) {
+			$('#pwd').val(pwd);
+		});
+	}
+	
     function initButton() {
         $('#reset-btn').click(function () {
             option.clear();
             initOptions();
+			initInput();
         });
+		$('#submit-btn').click(function () {
+			$("#suc").fadeIn();
+			setTimeout(function(){$("#suc").fadeOut();}, 2000);
+			if (!($('#checkbox2-7').prop('checked') && $('#checkbox2-8').prop('checked'))) {
+				return false;
+			}
+			let usr = $('#usr').val();
+			let pwd = $('#pwd').val();
+			if ((usr==='')||(pwd==='')) {
+				return false;
+			}
+			option.set('usr', usr)
+				.then(function () {
+					if (chrome.runtime.lastError) {
+						console.log(chrome.runtime.lastError);
+					}
+				});
+			option.set('pwd', pwd)
+				.then(function () {
+					if (chrome.runtime.lastError) {
+						console.log(chrome.runtime.lastError);
+					}
+				});
+		});
     }
 
     $(document).ready(function () {
         initVersion();
         initOptions();
+		initInput();
         initButton();
     });
 }(window, document, $));
```

Comment:
@fztfztfztfzt To make the code more clear, would you please replace `    ` (Tab) with `    ` (Space)? Thx.